utilities/tools/Clear-ManagementGroupDeployment.ps1

```diff
@@ -0,0 +1,96 @@
+﻿
+<#
+.SYNOPSIS
+Bulk delete all deployments on the given management group scope
+
+.DESCRIPTION
+Bulk delete all deployments on the given management group scope
+
+.PARAMETER ManagementGroupId
+Mandatory. The Resource ID of the Management Group to remove the deployments for.
+
+.PARAMETER DeploymentStatusToExclude
+Optional. The status to exlude from removals. Can be multiple. By default, we exclude any deployment that is in state 'running'.
+
+.EXAMPLE
```

Comment:
I'd suggest to add an example for `DeploymentStatusToExclude `usage

Reply:
done